kyaml/kio/byteio_reader.go

```diff
@@ -245,6 +260,44 @@ func (r *ByteReader) Read() ([]*yaml.RNode, error) {
 	return output, nil
 }
 
+// addSeqIndentAnno adds the sequence indentation annotation to the resource
+// value is the input yaml string and node is the decoded equivalent of value
+// the annotation value is decided by deriving the existing sequence indentation of resource
+func addSeqIndentAnno(value string, node *yaml.RNode) error {
```

Comment:
can we rename `value` to `origYAML`/`rawYAML` or something. value is too generic of a name.

Reply:
nit: consider putting `node` as the first argument to indicate the obj that is being updated here and rest are args to the operation.

Reply:
Updated.